block/submit.go

```diff
@@ -135,13 +135,12 @@ func (m *Manager) HandleSubmissionTrigger() error {
 		return fmt.Errorf("submit next batch to da: %w", err)
 	}
 
-	syncHeight, err := m.submitNextBatchToSL(nextBatch, resultSubmitToDA)
+	err = m.SLClient.SubmitBatch(nextBatch, m.DAClient.GetClientType(), resultSubmitToDA)
 	if err != nil {
-		return fmt.Errorf("submit pending batch to sl: %w", err)
+		return fmt.Errorf("sl client submit batch: start height: %d: inclusive end height: %d: %w", startHeight, endHeightInclusive, err)
 	}
 
-	// Update the syncTarget to the height of the last block in the last batch as seen by this node.
-	m.UpdateSyncParams(syncHeight)
+	m.UpdateSyncParams(endHeightInclusive)
```

Comment:
the created batch can be lower than `endHeightInclusive`

Reply:
ahh sorry I misread the code I didnt see this line
https://github.com/dymensionxyz/dymint/blob/48bda66be73fcf5c52497fc372961af9392521db/block/submit.go#L224
that explains a lot